sharedresource/v1alpha1/types_share.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright The OpenShift authors.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// Share is the Schema for the shares API
```

Comment:
Can I get a better description.  Who would create this and why?

Reply:
updated in next push

Reply:
I agree.  If I hadn't read the PR title or linked Jira issue, I'd have no idea this had to do with storage.  It isn't obvious what a Share represents, what consumes a Share, what happens when I create one, or where I'd start looking in the documentation to try to figure it out.